src/main/java/fi/aalto/cs/apluscourses/presentation/MainViewModelUpdater.java

```diff
@@ -28,10 +27,17 @@
   @NotNull
   private APlusProject aplusProject;
 
+  @NotNull
+  private final Project project;
+
```

Comment:
With this addition it might make sense to remove the `APlusProject` instance from this class, since none of the `APlusProject` "features" are actually used here anymore